moveit_core/CMakeLists.txt

```diff
@@ -162,8 +157,7 @@ include_directories(SYSTEM ${EIGEN3_INCLUDE_DIRS}
                            ${LIBFCL_INCLUDE_DIRS}
                            )
 
-#suppress spurious error: https://github.com/fkie/catkin_lint/issues/62
-#catkin_lint: ignore_once external_directory missing_directory  (${VERSION_FILE_PATH})
+#catkin_lint: ignore_once external_directory  (${VERSION_FILE_PATH})
```

Comment:
It looks like you removed the comment explaining and linking to the github issue, but kept the ignore_once - is that what you intended to do?

Reply:
(or was that issue only for the `missing_directory`)?

Reply:
Yes, the issue was related to `missing_directiory`. The `external_directoring` _warning_ is legitimate but should be suppressed here.